colossalai/inference/config.py

```diff
@@ -1,7 +1,6 @@
 """
-Our config consists of two parts:
+Our config consists of one part:
     1. inference_config: configs for inference, it is a unified api that wraps all the configs for inference.
```

Comment:
If it is only one part, you should just phrase like `Our config contains various options for inference optimization, it is a unified API that wraps all the configurations for inference.`

Reply:
Done